lib/cocoapods/deintegrator.rb

```diff
@@ -23,6 +23,9 @@ def deintegrate_target(target)
         deintegrate_shell_script_phase(target, 'Check Pods Manifest.lock')
         deintegrate_shell_script_phase(target, 'Embed Pods Frameworks')
         deintegrate_pods_libraries(target)
+        target.build_configurations.each do |config|
+          config.base_configuration_reference.remove_from_project
```

Comment:
Ah missed that :( Do you think matching file name with `/^Pods.*\.xcconfig$/i` is enough, like you do in `delete_pods_file_references` method?